routes/api.php

```diff
@@ -2,17 +2,16 @@
 
 use Illuminate\Http\Request;
 
-/*
```

Comment:
think we should leave this comment block so the file doesnt look way too different from the core file?

Reply:
Makes sense, re-added.